index.html

```diff
@@ -50,39 +49,64 @@ <h3>Furahia Insurance <br><span class="logo-writing2">Since 1965</span></h3>
       <h2 id="intro1">Enjoy life na Furahia Insurance!!</h2>
     </div>
   </header>
-  <section>
-    <article>    
-      <h1 id="intro2">PROMOTING A LIFE <br> FREE LIFESTYLE <br> 2022</h1>  
-      <p>Peace of mind comes from knowing you and your family are covered and protected by the best health insurance cover</p>
+  <main>
+    <article>
+      <h1 id="intro2">PROMOTING A LIFE <br> FREE LIFESTYLE <br> 2022</h1>
```

Comment:
- [ ] Kindly reduce the font size of the heading on mobile screens. It's way too bigger on mobile screens and the experience does not look nice. You must have forgotten about that, please fix that okay 👍

Reply:
@AmaduKamara This has now been fixed